.eslintrc

```diff
@@ -0,0 +1,26 @@
+{
+    "parser": "@typescript-eslint/parser",
+    "plugins": ["@typescript-eslint", "sort-imports-es6-autofix"],
+    "ignorePatterns": "*.html",
+    "rules": {
+        // Statements must end with semicolons (has autofix)
```

Comment:
can we have a comment that links to the documentation for this format? so we can change it?